### DISCUSSION TITLE
Intended user interface?

### DISCUSSION THREAD

**Discussion by atomtoto** [skiptools/skip#201] [Ideas]
Hello, is a user interface is planed for Skip or will it remain only by the terminal?

Thank you for this fantastic tools anyway!

**Comment by marcprux** [comment]
Nothing planned at this time. We generally expect that Xcode and Android Studio will be the primary interfaces for interacting with Skip's capabilities. Most of the tool operations are used for setup (like `skip checkup` and `skip init`), or are intended to be used headlessly from a CI machine (like `skip test`).
